src/YagrCore/plugins/plotLines/plotLines.ts

```diff
@@ -74,48 +79,11 @@ export default function plotLinesPlugin(options: PlotLineOptions): PlotLinesPlug
 
                 const {scale, value} = plotLineConfig;
 
-                if (Array.isArray(value)) {
-                    /** This weird code should handles unexpected Inifinities in values */
-                    const [fromValue, toValue] = value.map((val) => {
-                        if (Math.abs(val) !== Infinity) {
-                            if (scale === DEFAULT_X_SCALE) {
-                                if (val < timeline[0]) {
-                                    return timeline[0];
-                                }
-
-                                if (val > timeline[timeline.length - 1]) {
-                                    return timeline[timeline.length - 1];
-                                }
-                            } else {
-                                const scaleCfg = u.scales[scale];
-                                if (scaleCfg.min !== undefined && val < scaleCfg.min) {
-                                    return scaleCfg.min;
-                                }
+                const isBand = Array.isArray(value);
+                const [from, to] = calculateFromTo(value, scale, timeline, u);
```

Comment:
it's better to calculateFromTo only if isBand. is it?

Reply:
yep, made calculateFromTo shorter too)